### DISCUSSION TITLE
关于流媒体自动下一集

### DISCUSSION THREAD

**Discussion by Aurora-Simba** [hooke007/mpv_PlayKit#324] [Q&A 问答]
使用懒人包播放蜗牛云盘中阿里云盘视频时无法自动下一集，原版无问题

**Comment by hooke007** [comment]
我不使用该云盘所以无法提供对应复现方面的帮助。
关于原版无问题的判定来源是什么？纯净模式还是你自己的其它版本？如果是纯净模式则尝试修改部分设置去找到对应的引发问题的设置。如果是其他版本那引入的变量可能是mpv本身的版本问题而未必是配置问题。
